source/retryable-writes/tests/README.rst

```diff
@@ -330,7 +330,7 @@ and sharded clusters.
 
 #. Test that drivers properly retry after encountering PoolClearedErrors. This
    test MUST be implemented by any driver that implements the CMAP
-   specification.
+   specification. This test requires MongoDB 4.2.9+ for `blockConnection` support in the failpoint.
```

Comment:
RST uses double backticks for monospace formatting.

Reply:
fixed